### PR TITLE
Fixes for misc dbus methods

### PR DESCRIPTION
**Pull request by baude**
Ensuring that dbuss run, diff, and containerdelete all work.

### CODE REVIEW

File: Atomic/diff.py [hunk 1]
```diff
@@ -128,6 +128,7 @@ def create_image_list(images, args):
             try:
                 image_list.append(DiffObj(image, args))
             except mount.SelectionMatchError as e:
+                print(e)
```
Comment (rhatdan) [inline Atomic/diff.py]: debugging message?

File: atomic_dbus.py [hunk 2]
```diff
@@ -168,14 +172,15 @@ def ContainersList(self):
...
     @slip.dbus.polkit.require_auth("org.atomic.readwrite")
-    @dbus.service.method("org.atomic", in_signature='asbb', out_signature='')
-    def ContainersDelete(self, containers, all_containers, force):
+    @dbus.service.method("org.atomic", in_signature='asbbs', out_signature='')
+    def ContainersDelete(self, containers, all_containers, force, storage):
         c = Containers()
         args = self.Args()
-        args.container = containers[0]
-        args.containers = containers[1:]
+        print(type(containers))
```
Comment (rhatdan) [inline atomic_dbus.py]: Debugging?

File: atomic_dbus_client.py [hunk 3]
```diff
@@ -131,7 +137,7 @@ def Run(self, image, name=None, spc=False, detach=False, command=None):
             command = []
         if not isinstance(command, (list, tuple)):
             command = [ command ]
-        return self.dbus_object.Run(image, name, spc, detach, command, dbus_interface="org.atomic", timeout = 2147400)
+        return  self.dbus_object.Run(image, name, spc, detach, command, dbus_interface="org.atomic", timeout = 2147400)
```
Comment (rhatdan) [inline atomic_dbus_client.py]: Bogus space?

File: Atomic/diff.py (hunk 1 above)
Comment (baude) [inline Atomic/diff.py]: Yup, fixed

File: atomic_dbus.py (hunk 2 above)
Comment (baude) [inline atomic_dbus.py]: Yup fixed

File: atomic_dbus_client.py (hunk 3 above)
Comment (baude) [inline atomic_dbus_client.py]: Yup fixed.

**Comment by rhatdan** [comment]
@baude Tests are failing with an undefined e?

**Comment by rhatdan** [comment]
Needs a rebase.

**Comment by cgwalters** [comment]
bot, retest this please

**Comment by baude** [comment]
@rh-atomic-bot +877ff09d

**Comment by rhatdan** [comment]
@rh-atomic-bot r=rhatdan 877ff09d

**Comment by rhatdan** [comment]
@rh-atomic-bot r=rhatdan 87ff09d

**Pull request closed without merging**